db/migrate/20160311163203_change_data_type_for_column_vendor_in_vms_table_to_citext.rb

```diff
@@ -0,0 +1,9 @@
+class ChangeDataTypeForColumnVendorInVmsTableToCitext < ActiveRecord::Migration[5.0]
+  def up
+    change_column :vms, :vendor, :citext
```

Comment:
Correct, otherwise this would fail.
We could wrap this in something like [this](https://github.com/ManageIQ/manageiq/blob/master/lib/extensions/ar_adapter/ar_pglogical/pglogical_raw.rb#L10), but that will likely cause more problems than it will solve.